internal/db/queries.sql

```diff
@@ -33,8 +33,8 @@ WITH dequeued AS (
 	UPDATE mergestat.repo_sync_queue SET status = 'RUNNING'
 	WHERE id IN (
 		SELECT id FROM mergestat.repo_sync_queue
-		WHERE status = 'QUEUED'
-		ORDER BY repo_sync_queue.created_at ASC, repo_sync_queue.id ASC LIMIT 1 FOR UPDATE SKIP LOCKED
+		WHERE status = 'QUEUED' AND priority >=0
```

Comment:
```suggestion
		WHERE status = 'QUEUED'
```

I think we're okay to allow for negative priorities. If we don't, I think that should be a constraint on that column in the table for values going in, vs handling it here. But I don't see an issue (in fact it may be important) to allow for an "infinite upper bound" on priority in case we ever need to _ensure_ a job happened before all others